packages/dashmate/.eslintrc

```diff
@@ -4,6 +4,9 @@
     // Required for certain syntax usages
     "ecmaVersion": 2022
   },
+  "globals": {
+    "fetch": false
```

Comment:
Why this is set to false?

Reply:
Because fetch is now bundled with Node.js, and eslint errors about undefined function being used. The smarter way would be upgrade babel, eslint and the rest of the libraries to the recent version. I have already created a task for that

Reply:
https://github.com/dashpay/platform/issues/1296

Reply:
I see thank you